Islandora-CLAW/Crayfish
# Update lock files for crayfish using newer chullo

Fix Milliner tests relying on dead classes

**GitHub Issue**: n/a

Follow-up to https://github.com/Islandora-CLAW/Crayfish-Commons/pull/24 as Crayfish services all rely on `islandora/crayfish-commons:dev-master`. 

As a follow on to this (say not on a Friday) we probably should cut a `0.1` release of Crayfish-Commons and change all the services here from `islandora/crayfish-commons:dev-master` to `islandora/crayfish-commons:^0.1`

# What does this Pull Request do?

Updates lock files and fixes test fixtures.

# What's new?

Really updates the crayfish-commons version which updates the chullo version.

* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

I guess the best would be to pull this into a claw-playbook and see if it still works, but as only the tests still referred to the missing classes I think (🤞 ) we should be ok.

# Interested parties
@Islandora-CLAW/committers
